blocks/init/src/blocks/custom/featured-posts/components/featured-posts-options.js

```diff
@@ -3,24 +3,23 @@ import { __ } from '@wordpress/i18n';
 import { PanelBody, SelectControl, FormTokenField, RangeControl } from '@wordpress/components';
 
 export const postTypeOptions = [
```

Comment:
Did you leave this outside of the component on purpose? Is it being used in other places so it needs to be an exported const?

Reply:
this is used outside so it can be easily reused in storybook.
example:
https://github.com/infinum/wordpress-infinum-web/blob/master/wp-content/themes/infinum/src/blocks/components/button/components/button-options.js

https://github.com/infinum/wordpress-infinum-web/blob/master/wp-content/themes/infinum/src/blocks/components/button/docs/story.js